src/grid/CallLayout.ts

```diff
@@ -110,38 +109,33 @@ export function arrangeTiles(
   // use of screen space for n tiles without making those tiles too small or
   // too cropped (having an extreme aspect ratio)
   const gap = width < 800 ? 16 : 20;
-  const tileMinWidth = width < 500 ? 150 : 180;
+  const area = width * minHeight;
```

Comment:
width is the width and minHeight are for the whole area not just the tile right?